README.md

```diff
@@ -200,6 +200,10 @@ is raised. This module itself does not provide any threading
 capability, so if you need to consume the messages pushed from the
 server, you need to run it in a background thread.
 
+We provide 2 price data websockets. the polygon data, and the alpaca data.
```

Comment:
Can you capitalize the sentence correctly?

Reply:
done